docs/NXP_PXP_readme.md

```diff
@@ -0,0 +1,59 @@
+
+# HW acceleration for NXP iMX RT platforms using PXP (PiXel Pipeline) engine 
+
+Several drawing features in LVGL can be offloaded to PXP engine. In order to use CPU time while PXP is running, RTOS is required to block LVGL drawing thread and switch to another task, or simply to idle task, where CPU could be suspended to save power. 
+
+
+#### Features supported:
+- RGB565 format only
+- Area fill + optional transparency
+- BLIT (image transfer) + optional transparency
+- Color keying + optional transparency
+- Recoloring (color tint) + optional transparency
+- RTOS integration layer
+- default FreeRTOS and bare metal code provided
+
+#### Basic configuration:
+  - enable GPU support in lv_conf.h: Set LV_USE_GPU to 1
```

Comment:
`LV_USE_GPU` is used only to enable to old `gpu_fill_cb` and `gpu_blend_cb` in the display driver. It's really not clear from the comments in `lv_conf.h`. I'll update it. 

It seems your implementation doesn't rely on these functions so `LV_USE_GPU` can be `0`.

Reply:
Unfortunatelly some of the PXP code is inside `LV_USE_GPU` ifdef. If `LV_USE_GPU` is **only** for `gpu_fill_cb` and `gpu_blend_cb`, some PXP parts need to be moved out of `LV_USE_GPU` scope. I'll add fix.

Reply:
> I'll add fix.

Great, thanks!

Reply:
Added commit with fix - removed all LV_USE_GPU, so PXP is controlled only by PXP ifdefs. Regenerated internal conf. CI test passed now.

Reply:
Thank you!